### PR TITLE
[AIRFLOW-4445] mushroom cloud errors too verbose

### PR DESCRIPTION
**Pull request by tooptoop4**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Nice!

**Comment by potiuk** [comment]
Thanks @tooptoop4! One more request. Would you be so kind to update the AIRFLOW-XXX in the template (as requested in the template itself) of the PR to point to the right issue? It makes it a bit easier to resolve an issue after merging it (one-click from the description). We will soon add probot and we will be able to automate this part but until we have it, it would be great PR submitters do it.

**Comment by tooptoop4** [comment]
ok!

**Comment by potiuk** [comment]
BTW. Just proposed to simplify it in #6955 - feel free to comment :)

File: airflow/config_templates/default_airflow.cfg [hunk 1]
```diff
@@ -324,6 +324,12 @@ error_logfile = -
 # Expose the configuration file in the web server
 expose_config = False
 
+# Expose hostname in the web server
+expose_hostname = True
```
Comment (mik-laj) [inline airflow/config_templates/default_airflow.cfg:328]: Can you add some documentation about this option?   If there is no information about this feature in the documentation, very few people will be able to use it.

Comment (potiuk) [reply airflow/config_templates/default_airflow.cfg:328]: @mik-laj - > where do you think the documentation should be added ? Maybe you can provide some pointers? Most of the config options are documented in comments of the default_airlfow.cfg. Any other specific place you think this documentation should be added (and where all the previous options are documented?)

Comment (mik-laj) [reply airflow/config_templates/default_airflow.cfg:328]: I answered most of your questions in a different thread.:
https://github.com/apache/airflow/pull/6923#discussion_r362110552

I think we can add  new section in `security.rst`file. This file is long and suitable for refactoring, but I think we should do it in a separate PR. Please note that refactoring documentation, which is even a collection of random guides, is simpler than writing all documentation from scratch.

Comment (mik-laj) [reply airflow/config_templates/default_airflow.cfg:328]: A similar situation took place with the guides for operators.  Initially, we only had one file that contained a description of all operators. However, at some point, we refactored the documentation to create a new quality.

Comment (potiuk) [reply airflow/config_templates/default_airflow.cfg:328]: @tooptoop4 -> would you be so kind to add this in the doc section as explained by @mik-laj ?

**Pull request merged**